main.py

```diff
@@ -2,21 +2,16 @@
 # Date: 2024
 # Github: https://github.com/masterking32
 
+import asyncio
 import datetime
-import requests
```

Comment:
Why did you remove this line?

Reply:
it was imported in utilities but not used. moved to main